src/ir/item.rs

```diff
@@ -530,6 +530,11 @@ impl Item {
         &mut self.kind
     }
 
+    /// Where in the source is this item located?
+    pub fn location(&self) -> &Option<clang::SourceLocation> {
```

Comment:
Let's return an `Option<&clang::SourceLocation>` and use `self.location.as_ref()` instead, as that's a bit more idiomatic.

Reply:
Done.